### PR TITLE
FW-850 Adapt timing for openmote-b-subGHz board 

### PR DESCRIPTION
**Pull request by minarady1**

### CODE REVIEW

**Comment by changtengfei** [comment]
@minarady1  for the future work, when you commit codes, use the issue number at beginning so JIRA can hook up the changes. Such FW-850. 160 ms timing.  The branch name should be named as develop_FW-50.

**Comment by changtengfei** [comment]
@minarady1  Could you write a summary about how you test the code? When I check the radio_settings of subghz port, only one frequency is used. When using 11 length slotframe with 10 EB portion. The mote should be sync'ed within 16 second, which is not what I saw.

Please add a readme file record what needs to do  to see the same thing on your side.

**Comment by minarady1** [comment]
For this PR to be tested and to work correctly, the following needs to be configured. With this configuration, it should synchronise within 2 minutes and you should see the first RPL DAO within 10 mins.

Configuration > Location:
  LENGTH_CRC = 2 > radio.h (this is mandatory)
  fix_channel = <channel_id> scons compile option fix_channel = 11 (for example)
  EB_PORTION 4 > IEEE802154E.h
  SLOTFRAME_LENGTH11 > schedule.h

**Comment by changtengfei** [comment]
@minarady1 I had tested your code with the changes you mentioned,  it works on my side perfectly, with little changes. Could I ask to do the following before I merge your code?
- create a README.md file under project->openmote-b-subghz to record the changes to make to see the network starts up quickly.
- change Port_delayTx 100 to Port_delayTx 68
- change the timing in IEEE802154E.h file under slot duration 160 from ticks to micro-second/PORT_US_PER_TICK.

File: projects/openmote-b-subghz/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+* Go to bsp/chips/at86rf215/radio.h then make sure LENGTH_CRC is configured to 2
+
+
+To speed up the synchronization for this release (for testing and debugging purposes), make sure to apply the following modifications:
+
+Configuration | Location
+------------ | -------------
+fix_channel | add scons compile option fix_channel = 11 (for example)
+EB_PORTION 4 | openstack\02a-MAClow\IEEE802154E.h
+SLOTFRAME_LENGTH11 | openstack\02b-MAChigh\schedule.h
```
Comment (changtengfei) [inline projects/openmote-b-subghz/README.md]: A space is missing here

**Pull request merged**